test/ad/utils.jl

```diff
@@ -1,32 +1,36 @@
 # Figure out which AD backend to test
 const AD = get(ENV, "AD", "All")
 
-function test_ad(f, x, broken=(); rtol=1e-6, atol=1e-6)
-    finitediff = FiniteDifferences.grad(central_fdm(5, 1), f, x)[1]
+function test_ad(f, x, broken=(); rtol=1e-6, atol=1e-6, use_forwarddiff_as_truth=false)
+    truth = if use_forwarddiff_as_truth
```

Comment:
Why is this needed, why does finite differencing fail? Because the positive-definiteness is violated by perturbations? The common fix (also used in DistributionsAD) for such problems is to test AD of a different function, e.g., `g(A) = f(A * A' + I)` instead of `f` directly.

Reply:
Ah, of course! Cheers!